src/app/workflow/workflow.component.html

```diff
@@ -80,11 +80,21 @@ <h3 id="workflow-path" data-cy="workflowTitle" fxLayout="row" fxLayoutAlign="sta
         </mat-form-field>
       </div>
       <p class="update">
-        <span [matTooltip]="(extendedWorkflow$ | async)?.last_modified_date | date: 'medium'"
-          >Last Modified: {{ (extendedWorkflow$ | async)?.agoMessage || 'n/a' }}</span
-        >
+        <span *ngIf="selectedVersion !== null">
+          <span *ngIf="'TAG' === selectedVersion.referenceType" [matTooltip]="selectedVersion.last_modified | date: 'medium'"
```

Comment:
There is an enum, ReferenceTypeEnum, you could use instead of `TAG`.

Reply:
Running into 
https://stackoverflow.com/questions/35835984/how-to-use-a-typescript-enum-value-in-an-angular2-ngswitch-statement
I can declare a reference in `workflow.component.ts` but it doesn't seem like a very good way of doing it

Reply:
Looks like we do it already: https://github.com/dockstore/dockstore-ui2/blob/21b3959c8f21202a8be4e3cb500a16028d2b4b7c/src/app/workflow/workflow.component.ts#L95

But I don't feel super strongly about this.

Reply:
I'm ok with it, if its established practice :grin: